two_factor_otp_auth/models/res_users.py

```diff
@@ -99,7 +99,6 @@ def _check_credentials(self, password):
             `otp_code` in request params. Should be caught by controller and
             render and open enter "one-time-password" page or QR code creation
         """
-        self.ensure_one()
```

Comment:
@ploegvde , hm...  Very interesting why we didn't catch it ourselves.
I see standard Odoo 13.0 are not there https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/res_users.py#L297 

So merging this